src/WebUI/dotnet/WebPortal/Views/Home/JobSubmission.cshtml

```diff
@@ -1,1036 +1,1006 @@
-                            <md-select name="jobtrainingtype" ng-model="extras.preemptionAllowed" required md-no-asterisk>
-                                <md-option value="False" selected>Non-Preemptible Job</md-option>
-                                <md-option value="True">Preemptible Job</md-option>
-                            </md-select>
-                        </md-input-container>
-                    </div>
-
-                    <div layout="row" ng-if="current.jobtrainingtype=='PSDistJob'">
-
-                        <md-input-container flex="50">
-                            <label> Number of Workers </label>
-                            <input ng-model="current.numpsworker" required type="number" step="1" min="1" md-no-asterisk>
-                        </md-input-container>
-
-                        <md-input-container flex="50">
-                            <label> Total number of GPUs </label>
-                            <input ng-value="current.numpsworker * gpumeta[extras.gpuType]['num_gpu_per_node']" disabled type="number" md-no-asterisk>
-                        </md-input-container>
-
-                    </div>
-
-                    <md-input-container class="md-block" ng-if="current.jobtrainingtype=='RegularJob'">
-                        <label> Number of GPUs  Per Worker</label>
-                        <input ng-model="current.resourcegpu" required type="number" name="resourcegpu" step="1" min="0" max="{{gpumeta[extras.gpuType]['num_gpu_per_node']}}" md-no-asterisk>
-                        <div ng-messages="jobApp.resourcegpu.$error" multiple>
-                            <div ng-message="required">
-                                You have to let us know the number of gpus per worker.
-                            </div>
-
-                            <div ng-message="min">
-                                You have to request at least 0 GPU per worker.
-                            </div>
-
-                            <div ng-message="max">
-                                You can request at most {{gpumeta[extras.gpuType]['num_gpu_per_node']}} GPUs per worker.
-                            </div>
-                        </div>
-                    </md-input-container>
-
-
-
-                    <md-input-container class="md-block">
-                        <label> Docker Image </label>
-                        <input ng-model="current.image" required md-no-asterisk>
-                    </md-input-container>
-
-                    <md-input-container class="md-block">
-                        <label>Command</label>
-                        <textarea ng-model="current.cmd" rows="10"></textarea>
-                    </md-input-container>
-
-                    <div layout="row">
-                        <md-input-container class="md-block" flex="50">
-                            <md-switch ng-model="extras.ssh" md-no-asterisk>
-                                SSH

```

Comment:
Why are these switches removed in this PR?

Reply:
I reverted this merge. Can you double check if there are any changes are missed here?